tests/integration/local/secrets/store_test.go

```diff
@@ -117,3 +118,32 @@ func Test_IncorrectSecretDataKey(t *testing.T) {
 	})
 
 }
+
+func TestCreate_Secret(t *testing.T) {
+	nsName := tests.CreateNamespace(t)
+	k8sConfig := secrets.PluginConfig{
+		Namespace: nsName,
+		Logger:    logger,
+	}
+	tc := portercontext.TestContext{}
```

Comment:
Is there a reason you aren't using portercontext.NewTestContext()? This skips all the logic and configuration when making a test context.

Reply:
I don't think so. I changed it to use the constructor